docs/how-to-guides/how-to-setup-triggers.md

```diff
@@ -0,0 +1,101 @@
+
+### Trigger execution upon node creation
+
+Node creation can be most common event your Memgraph database can react to. For example, you need to updated some value on created node. 
+If you need a trigger after the node(vertex) has been created, you can set up the following trigger: 
+
+```cypher
+CREATE TRIGGER createVertex
+ON () CREATE AFTER COMMIT EXECUTE
+UNWIND createdVertices AS createdVertex
+SET createdVertex.created = timestamp()
+```
+
+Here the trigger's name is `createdVertex` and should be unique. Cypher command  `ON CREATE` defines on what event will trigger be executed. `AFTER COMMIT EXECUTE` means the trigger will be executed after changes have been committed to the database. For the sake of ease of use, triggers have a set o **predefined variables**. One is `createdVertices`, a list of all created nodes (vertices). Here the list is being unwinding by cypher command `UNWIND` 
+To find a complete list of predefined variables, supported operations, and configuration details, look at triggers [reference guide](https://memgraph.com/docs/memgraph/reference-guide/triggers).
+
+In this trigger via `SET createdVertex.created = timestamp()`  command, new node is getting a timestamp after creation.
+
+### Trigger execution upon node update
+
+Node property updates are common in graphs,in order to react to them you can create trigger for that type of event: 
+
+```cypher
+CREATE TRIGGER updateVertex
+ON () UPDATE AFTER COMMIT EXECUTE
+UNWIND updatedVertices AS updatedVertex
+SET updatedVertex.updated_at = timestamp()
+
+```
+The trigger for node updates is almost the same as node creation. Notice different predefined variable `updatedVertices` and `ON UPDATE` cypher command.
+
+In this trigger via `SET updatedVertex.updated_at = timestamp()` command, node is getting new updated timestamp.
+
+### Trigger execution upon node or relationship creation
+
+You can also setup the trigger for multiple events, such as node or relationships creation. It does matter what will be created node or relationship, but it will execute the trigger.
+Here is the sample query for the trigger: 
+
+```cypher
+CREATE TRIGGER exampleTrigger
+ON CREATE AFTER COMMIT EXECUTE
+UNWIND createdObjects AS createdObject
+WITH CASE
+        WHEN createdObject.vertex IS NOT null THEN createdObject.vertex
+        WHEN createdObject.edge IS NOT null THEN createdObject.edge
+    END AS object
+SET object.created_at = timestamp();
+```
+
+By using predefined variable `createdObjects` you are getting all created objects. Variable `createdObjects` is a list of maps where each map contains `event_type` and `edge` or `vertex` property. Both properties depends on 
+what type of object has been created. If your Query is creating multiple nodes and relationships, they will be in a separate created objects maps. 
+
+In this trigger via `SET object.created_at = timestamp();` command, node or relationships property `created_at` is set to current timestamp. 
```

Comment:
In this trigger, the node or relationships property `created_at` is set to the current timestamp via bla bla.